01week/rockPaperScissors.js

```diff
@@ -9,15 +16,32 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
+  // Write code here 
+  const cleanHand1 = hand1.toLowerCase().trim(); //re-assign the input value to a variable if scrubbing. 
+  const cleanHand2 = hand2.toLowerCase().trim();  
+  if(cleanHand1 === 'rock' || cleanHand1 === 'paper' || cleanHand1 === 'scissors' && cleanHand2 === 'rock' || cleanHand2 === 'paper' || cleanHand2 === 'scissors'){
```

Comment:
this is a bit redundant, what would be a better way to check if the input is valid?